src/tools/cephfs/cephfs-shell

```diff
@@ -733,6 +733,11 @@ exists.')
                     self.poutput(path, end=':\n')
                 items = sorted(ls(path),
                                key=lambda item: item.d_name)
+            if items == []:
+                self.perror('{}: no such directory exists'.format(path),
+                            end='\n', apply_style=True)
```

Comment:
`perror` doesn't take arguments like `end` and `apply_style`.

Reply:
According to the [cmd2 doc:](https://cmd2.readthedocs.io/en/latest/api/cmd.html?highlight=perror#cmd2.cmd2.Cmd.perror)
```
perror(msg: Any, *, end: str = '\n', apply_style: bool = True) 
```

Reply:
Ok. My cmd2 module wasn't latest. The new version supports it.